clusters/build-clusters/01_cluster/cert-manager/cert-manager.yaml

```diff
@@ -7059,6 +7059,7 @@ spec:
           - --v=2
           - --cluster-resource-namespace=$(POD_NAMESPACE)
           - --leader-election-namespace=kube-system
+          - --dns01-recursive-nameservers="8.8.8.8:53"
```

Comment:
This doesn't go through a shell so I think you will have to remove the quotes

Reply:
It works with build02 deployment, though.